cypress/integration/Chip/Accepts_children.feature

```diff
@@ -0,0 +1,5 @@
+Feature: The Chip renders children
```

Comment:
For the chip it's important to test the `onRemove` prop. I.e. if that prop is supplied a "remove icon" is rendered and `onRemove` should be called when that icon is clicked.

Reply:
Definitely. That test already existed though: https://github.com/dhis2/ui-core/blob/3a8db74bc13f30cbe1b5f58e02c494de5cc363ea/cypress/integration/Chip/Is_removable.feature. Let me know if you think it's sufficient.

Reply:
Aha, I didn't realise that. I'll mark this as resolved.